app/javascript/angular/code/controllers/_mobile_sessions_map_ctrl.js

```diff
@@ -174,6 +176,12 @@ export const MobileSessionsMapCtrl = (
 
         FiltersUtils.fetchShortUrl(tooltipId, currentUrl);
       });
+
+      elmApp.ports.toggleTheme.subscribe(isCustomThemeOn => {
+        params.update({ customTheme: isCustomThemeOn });
+        $scope.$apply();
+        applyTheme();
+      });
```

Comment:
Just as a random thought, could we merge `_mobile_sessions_map_ctrl.js ` and `_fixed_sessions_map_ctrl.js`, add some ifs and pass from Elm the `Page` the app is on? Not sure if that would add too many ifs but it would get rid of the double controller file